composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/renovate.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Just for my own knowledge, what does this file do? :)

Reply:
Great question! This is a configuration file for [whitesource renovate bot](https://www.whitesourcesoftware.com/free-developer-tools/renovate/), our robot bff that watches our repos and looks for dependencies that need to be updated